vector/src/main/java/im/vector/app/features/roomprofile/polls/RoomPollsFilter.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.roomprofile.polls
+
+enum class RoomPollsFilter {
```

Comment:
We generally add `Type` suffix for the enum class names. What do you think about renaming as `RoomPollsFilterType`?

Reply:
Agree, updated in [d604035](https://github.com/vector-im/element-android/pull/7867/commits/d60403545c4f323f7067bef06776660cdcf4b4d3).